tests/python/gaia-ui-tests/gaiatest/apps/ftu/app.py

```diff
@@ -334,9 +334,20 @@ def a11y_click_next_to_welcome_browser_section(self):
             Wait(self.marionette).until(expected.element_present(
                 *self._section_welcome_browser_locator))))
 
+    @property
+    def is_statistics_enabled(self):
+        return self.marionette.find_element(
+            *self._statistic_checkbox_locator).is_selected()
+
     def tap_statistics_checkbox(self):
```

Comment:
This function does the same thing as the one you created (without the check though). I would remove it and update the other place where this function is used (test_ftu_skip_tour_for_desktop).